srcpkgs/ampache/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'ampache'
 pkgname=ampache
-version=5.5.1
+version=5.6.2
 revision=1
 create_wrksrc=yes
 hostmakedepends="unzip"
-depends="php8.1 mariadb"
+depends="virtual?php-runtime"
```

Comment:
what about mariadb?

Reply:
As far as I know ampache doesn't require the database to be on the same machine as ampache as it can be configured with [`database_hostname`](https://github.com/ampache/ampache/blob/develop/config/ampache.cfg.php.dist#L78).